tests/tr1/tests/cmath1/test.cpp

```diff
@@ -50,6 +52,8 @@ void test_cpp() { // test C++ header
 
     double (*pabs)(double) = &STD abs; // C++ overloads
     CHECK((*pabs)(-rthalf) == rthalf);
+
+    STATIC_ASSERT(std::is_same<decltype(std::pow(10.0, 2)), double>::value);

```

Comment:
Thanks for adding test coverage! However, as https://github.com/microsoft/STL/blob/master/tests/tr1/README.md explains, we generally don't add anything to `tr1` these days. (We'll update it for breaking changes.)

There doesn't seem to be a super great existing test in `tests/std` to add such coverage; `VSO_0000000_instantiate_containers` tests `pow` for `complex` so we could add separate tests for arithmetic `pow`. Alternatively, adding a new test directory would be reasonable.

In addition to these test cases which are exercising `(double, int)`, `(float, int)`, and `(long double, int)`, I'd like to see more extensive coverage (as the "sufficient additional overloads" wording applies to the first argument being integral too). For example, this is a comprehensive test for `complex` `pow`; the arithmetic cases would be strictly fewer:

https://github.com/microsoft/STL/blob/0cdf5fbfac83650bf6b404a754be415493466643/tests/std/tests/VSO_0000000_instantiate_containers/test.cpp#L866-L918

Reply:
Tests added.